draft-ietf-trans-rfc6962-bis.md

```diff
@@ -656,10 +656,15 @@ previously logged as a precertificate, then the precertificate's SCT of type
 `precert_sct_v2` would not be appropriate; instead, a fresh SCT of type
 `x509_sct_v2` should be generated.
 
-An SCT is the log's promise to incorporate the submitted entry in its Merkle
-Tree no later than a fixed amount of time, known as the Maximum Merge Delay
-(MMD), after the issuance of the SCT. Periodically, the log MUST append all its
-new entries to its Merkle Tree and sign the root of the tree.
+An SCT is the log's promise to append, to its Merkle Tree, an entry for the
+submission, no later than a fixed amount of time (known as the Maximum Merge
+Delay, MMD), after the issuance of the SCT. An entry for a submission is
```

Comment:
My attempt at rewording this sentence with fewer commas:
"An SCT is the log's promise to append to its Merkle Tree an entry for the accepted submission no later than a fixed amount of time, known as the Maximum Merge Delay (MMD), after the issuance of the SCT."